### DISCUSSION TITLE
Version 1.3.0

### DISCUSSION THREAD

**Discussion by Adam777Z** [Adam777Z/save-private-window-cookies-web-browser-extension#4] [Announcements]
- Manifest updated
- Updated for Firefox first-party isolation
- Options Dark Mode support

<hr /><em>This discussion was created from the release <a href='https://github.com/Adam777Z/save-private-window-cookies-web-browser-extension/releases/tag/1.3.0'>Version 1.3.0</a>.</em>

**Comment by Lolguy1** [comment]
Hi, im sorry to bother but im having some issues with this extension. Im currently using Chromite as my browser aka a fork of Bromite which itself is a fork of Chromium and when I try to use this extension, in Incognito it says its blocked by the client even after disabling my ad blocker and allowing the extension in incognito. I have to manually go to chrome-extension:// and enable dev settings to enable "Collect errors" and "Allow access to file urls" which THEN allows me to access it, so then I login to some Google accounts and other websites and it DOES detect these cookies. It has the option to backup and restore but strangely the top option to Save cookies is greyed out. I thought it was fine and made it save the Cookies. Keep in mind they were originally 12KB when I backed them up and checked the file size but when I restored them after I backed them up just to make sure it worked it said it was 9KB which was strange, so I checked my Google accounts and only ONE Google account remained! The rest of the Google accounts were gone and I was wondering what went wrong. If you can please test this on a mobile device with Chromite on it and enable the option to have extensions on mobile in its Developer Options, and do what I did and try to fix the issue in your next update. I logged into 3 Google accounts btw. Thank you for making this extension! :)
